### PR TITLE
Voice Broadcast playback UI

### PR DESCRIPTION
**Pull request by weeman1337**
Part of https://github.com/vector-im/element-web/issues/23282

- Voice Broadcast playback model, store and basic UI
- The actual playback is part of another PR

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/voice-broadcast/components/VoiceBroadcastBody.tsx [hunk 1]
```diff
@@ -15,24 +15,44 @@ limitations under the License.
...
+    VoiceBroadcastPlaybackEvent,
+    VoiceBroadcastPlaybackState,
+    VoiceBroadcastPlayback,
+    VoiceBroadcastPlaybackBody,
 } from "..";
 import { IBodyProps } from "../../components/views/messages/IBodyProps";
 import { MatrixClientPeg } from "../../MatrixClientPeg";
 import { useTypedEventEmitter } from "../../hooks/useEventEmitter";
 
+// TODO Michael W: refactor component to separate ones or hooks or…
```
Comment (weeman1337) [inline src/voice-broadcast/components/VoiceBroadcastBody.tsx:38]: Already prepared and will follow soon

File: src/voice-broadcast/models/VoiceBroadcastPlayback.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    public getState(): VoiceBroadcastPlaybackState {
+        return this.state;
+    }
+
+    private setState(state: VoiceBroadcastPlaybackState): void {
+        this.state = state;
+        this.emit(VoiceBroadcastPlaybackEvent.StateChanged, state);
+    }
+
+    destroy(): void {
```
Comment (turt2live) [inline src/voice-broadcast/models/VoiceBroadcastPlayback.ts:73]:
```suggestion
    public destroy(): void {
```

**Review by turt2live — APPROVED** [review]
This looks fine enough for now - we might want to revist some of the css to reuse between voice messages and other similar elements, but that's relatively minor.

**Comment by turt2live** [comment]
The login test has passed, so that's good. Force merging given the remainig test is legitimate as a failure but not because of this PR.

**Pull request merged**